src/game.cpp

```diff
@@ -1688,7 +1688,75 @@ ReturnValue Game::internalTeleport(Thing* thing, const Position& newPos, bool pu
 	return RETURNVALUE_NOTPOSSIBLE;
 }
 
+Item* searchForItem(Container* container, uint16_t itemid)
```

Comment:
You should rename the parameter ``itemid`` to ``itemId``.
You should also rename the function name to (since ``searchForItem`` alone implies a ``void`` function, etc...) ``getContainerItem`` (since it actually ``return``s a pointer to the ``Item`` inside the container that matches the item id you're comparing to).